packages/flutter/lib/src/material/date_picker_theme.dart

```diff
@@ -252,6 +253,19 @@ class DatePickerThemeData with Diagnosticable {
   /// or pressed.
   final MaterialStateProperty<Color?>? yearOverlayColor;
 
+  /// Overrides the default shape used to paint the shape decoration of the
+  /// year labels in the list of the year picker.
+  ///
+  /// If the selected year is the current year, the provided shape with the
+  /// value of [todayBackgroundColor] is used to paint the shape decoration of
+  /// the year label and the value of [todayBorder] and [todayForegroundColor] is
```

Comment:
How are `todayBackgroundColor` and `todayForegroundColor` used in the code above? I only see `todayBorder` and `textColor`.